webapp/components/ReleaseModal/ReleaseModal.spec.js

```diff
@@ -0,0 +1,183 @@
+import {
+  shallowMount,
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-05-27T06:50:21Z" title="Monday, May 27th 2019, 8:50:21 am +02:00">May 27, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,150 @@
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
+import DisableModal from './ReleaseModal.vue'
```


Should be named `ReleaseModal`instead `DisableModal `, I think.